src/host/screenInfo.hpp

```diff
@@ -305,6 +309,9 @@ class SCREEN_INFORMATION : public ConsoleObjectHeader, public Microsoft::Console
 
     bool _ignoreLegacyEquivalentVTAttributes;
 
+    std::map<SHORT, std::wstring> _hyperlinkMap;

```

Comment:
Unless you particularly need an ordered map here, note that `std::unordered_map` is generally more efficient.

Reply:
Fixed!